01-data-structures/01-introduction-to-data-structures/screen/pixel.rb

```diff
@@ -11,11 +11,22 @@ class Pixel
 
 
   def initialize(red, green, blue, x, y)
+    self.red = validate_color(red)
+    self.green = validate_color(green)
+    self.blue = validate_color(blue)
+    self x = x
+    self y = y
```

Comment:
Looks like you are missing `.` for the `x` and `y` initialization. I am guessing Ruby still let this work, because Ruby is strange like that. I tend to use the `@` syntax for initialization: `@x = x`. Using `self.x = x` will actually execute the `x=` method, which may or may not be desired. 99% of the time it makes no difference. However, if you do end up putting a custom `x=` method on your class, that code will execute it instead of setting the instance variable directly. Whether or not that is the desired behavior is up to you.